docs/package_integration/uds-package-practices.md

```diff
@@ -120,12 +112,30 @@ In practice, this results in the following for the second release of a package f
 17.2.1-uds.1
 ```
 
-## General
+## Documenting Exceptions
+
+If you are unable to meet a requirement or best practice, document the justification in your package repository `docs/package_exceptions.md`
+
+Some examples of valid justifications would be:
+- ✅ The last step of deployment requires a click-ops interaction and cannot be automated through IaC
+- ✅ Upstream application does not expose metrics endpoint to integrate
```

Comment:
IMO this one is also odd since the wording `- **Must** implement monitors for each application metrics endpoint using it's built-in chart monitors, "monitor" key, or manual monitors in the config chart.` would not apply if the application doesn't implement monitors - there would be no for each loop to check through - also are we documenting exceptions to Musts or other parts with flexibility (Shoulds/Mays)?  IMO Musts **must** be the things that we will not compromise on and if there are exceptions (i.e. no user login, no metrics endpoint) those should simply be worded into and called out in the Must text.

Reply:
removed this part :)